scripts.v3/utils.js

```diff
@@ -418,13 +406,43 @@ class ImporterExporter {
             }
 
             await this.httpClient.sendRequest("PUT", url, body);
-        }
-        catch (error) {
+        } catch (error) {
             throw new Error(`Unable to schedule website publishing. ${error.message}`);
         }
     }
+
+    /**
+     * Updates the url links of specified API Management service into updated url in destination.
+     */
+    async updateContentUrl(existingUrls, replaceableUrls) {
+        try {
+            const contentItems = await this.getContentItems("url");
+
+            console.log("urls found in portal: " + contentItems.length);
+
+            for (const contentItem of contentItems) {
+                var count = 0;
+                console.log("url found in portal: " + contentItem.properties.permalink);
+
+                for (const existingUrl of existingUrls) {
+                    if (contentItem.properties.permalink == existingUrl) {
+                        contentItem.properties.permalink = replaceableUrls[count];
+                        console.log("updating url content..." + contentItem.properties.permalink);
+                        console.log(" updated url content : " + JSON.stringify(contentItem));
+                        const response = await this.httpClient.sendRequest("PUT", contentItem.id + "?api-version=2019-12-01", contentItem);
```

Comment:
Please use `apiVersion` constant defined above in this file rather the in-place version.

Reply:
Created a variable.